css-color-5/Overview.bs

```diff
@@ -684,10 +684,13 @@ Selecting the most contrasting color: the ''color-contrast()'' function {#colorc
 
 	<h3 id="resolving-contrast">Resolving ''color-contrast()'' values</h3>
 
-	The computed and used value is the specified ''color-contrast()'' function
+	The computed value is the specified ''color-contrast()'' function
 	with each <<color>> parameter resolved according to [[css-color-4#resolving-color-values]],
 	and the keywords ''AA'', ''AA-large'', ''AAA'', ''AAA-large'' replaced with their corresponding numeric value.
 
+	The used value is the winning color
+	resolved according to [[css-color-4#resolving-color-values]].
```

Comment:
That implies it is always possible to determine the winning color. Is it? Even with custom properties, calc and so on in the mix?

Reply:
By used-value time, yes. Custom properties disappear at computed-value time; calc()s disappear, at their latest, at used-value time as well. (Everything *better* be resolvable by then, since CSS doesn't actually *define* anything happening past that, just that the UA can make unstated modifications to obtain the "actual value".)